tests/parsers/unchained_labs_lunatic/testdata/2024-07-12_CW288_Plate1.json

```diff
@@ -16,6 +16,7 @@
                                 "device control document": [
                                     {
                                         "device type": "plate reader",
+                                        "detection type": "Absorbance",
```

Comment:
NOTE to reviewer: these changes are because unchained parser previously wasn't adding detection type (because it was implied) but no harm in adding it, for consistency.